Userland/Applications/TaskbarSettings/TaskbarSettingsMain.gml

```diff
@@ -0,0 +1,52 @@
+@GUI::Widget {
+    fill_with_background_color: true
+
+    layout: @GUI::VerticalBoxLayout {
+        margins: [10]
+        spacing: 5
+    }
+
+    @GUI::GroupBox {
+        title: "Taskbar Appearance"
+        shrink_to_fit: false
+        fixed_height: 165
+
+        layout: @GUI::VerticalBoxLayout {
+            margins: [16, 8, 8]
+            spacing: 16
+        }
+
+        @GUI::Label {
+            text: "Location of taskbar on screen"
+            text_alignment: "TopLeft"
+        }
+
+        @GUI::Widget {
+            shrink_to_fit: true
+
+            layout: @GUI::VerticalBoxLayout {
```

Comment:
Wouldn't it be better to have this as a `HorizontalBoxLayout`? In your screenshot all the radios are just on the left, not making very good use of the available space.